### PR TITLE
add schema for model update

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

**Review by homezcx — APPROVED** [review]

File: rlclientlib/schema/v2/ModelUpdateEvent.fbs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace reinforcement_learning.messages.flatbuff.v2;
+
+table ModelUpdate {
+    // VW ModelDelta
+    delta: [ubyte];
```
Comment (rajan-chari) [inline rlclientlib/schema/v2/ModelUpdateEvent.fbs:5]: Is this not just a collection of (hash, float) values?

Comment (jackgerrits) [reply rlclientlib/schema/v2/ModelUpdateEvent.fbs:5]: No a VW ModelDelta is pretty much a full VW model. This is to adhere to the model merging operations defined in VW.

Comment (rajan-chari) [reply rlclientlib/schema/v2/ModelUpdateEvent.fbs:5]: To be specific, my concern is around versioning.  If this is an opaque payload that changes over time, we would then have to worry about cases where the sender and reciever might be out of sync. Where as being specific about schema means we can simply send a different message that's either handled or not.

Comment (rajan-chari) [reply rlclientlib/schema/v2/ModelUpdateEvent.fbs:5]: Ah ok.  makes sense.  Thanks

**Review by rajan-chari — APPROVED** [review]
Looks good thanks.

File: rlclientlib/schema/v2/ModelUpdateEvent.fbs (hunk 1 above)
Comment (jackgerrits) [reply rlclientlib/schema/v2/ModelUpdateEvent.fbs:5]: This work is still experimental, so I have we can update to consume the schematized VW model so we won't need to worry.

For now though it follows the current "status quo" of VW models, so its version is embedded within it and as long as the format doesn't change should be loadable.

**Pull request merged**